docs/hive_metrics.md

```diff
@@ -127,4 +128,20 @@ Some of these metrics are optional and the admin can opt for logging them via `H
 |               hive_controller_reconcile_seconds                |           N            |    N     |
 |             hive_cluster_deployment_syncset_paused             |           N            |    N     |
 |       hive_cluster_deployment_provision_underway_seconds       |           N            |    N     |
-|  hive_cluster_deployment_provision_underway_install_restarts   |           N            |    N     |
\ No newline at end of file
+|  hive_cluster_deployment_provision_underway_install_restarts   |           N            |    N     |
+
+### Example: Configure metricsConfig
+
+```sh
+oc edit hiveconfig -n hive
+```
+
+```yaml
+spec:
+  metricsConfig:
+    metricsWithDuration:
+      - name: currentClusterSyncFailing
```

Comment:
Since we are adding more documentation around this, we might as well mention what are the acceptable values (or atleast link them to the right location)

Reply:
+1. I.e. consts of type `DurationMetricType` in duration_metrics.go.